Protocol.md

```diff
@@ -467,18 +467,32 @@ Frame Contents
     |                         Stream ID = 0                         |
     +-----------+-+-+-+-------------+-------------------------------+
     |Frame Type |0|0|R|    Flags    |
-    +-----------+-+-+-+-------------+-------------------------------+
-    |0|                  Last Received Position                     |
-    +                                                               +
+    +-----------+-+-+-+-------------+---------------+---------------+
+    |0|0|0|0|           Number of Entries           |               
+    +-------------------------------+---------------+---------------+
+    |0|                         Stream ID                           |
+    +-------------------------------+-------------------------------+
     |                                                               |
-    +---------------------------------------------------------------+
+    |                       Implied Position                        |
+    |                                                               |
+    +-------------------------------+-------------------------------+
+    |0|                         Stream ID                           |
+    +-------------------------------+-------------------------------+
+    |                                                               |
+    |                       Implied Position                        |
+    |                                                               |
+    +-------------------------------+-------------------------------+
+    |                              ...                              |
+    +-------------------------------+-------------------------------+
                                   Data
 ```
 
 * __Frame Type__: (6 bits) 0x03
 * __Flags__: (10 bits)
      * (__R__)espond with KEEPALIVE or not
-* __Last Received Position__: (63 bits = max value 2^63-1) Unsigned 63-bit long of Resume Last Received Position. Value MUST be > 0. (optional. Set to all 0s when not supported.)
+* __Number of Entries__: (24 bits = max value 2^20 = 1,048,576) (optional. Set to all 0s when not supported.)
+* __Stream ID__: (31 bits = max value 2^31-1 = 2,147,483,647) Unsigned 31-bit integer representing the resumable stream Identifier. (present if Number of Entries > 0)
```

Comment:
Number of entries is described as 24 bits but in reality it is always preceded by 4 `0`s, meaning the max value is `2^20` as correctly stated above. For fields like stream id it will be documented as the number of bits that are significant, in this case 20 (stream id takes 32 bits in the frame but always has a leading `0`, so it says 31 bits in the spec).
```suggestion
* __Number of Entries__: (20 bits = max value 2^20 = 1,048,576) (optional. Set to all 0s when not supported.)
* __Stream ID__: (31 bits = max value 2^31-1 = 2,147,483,647) Unsigned 31-bit integer representing the resumable stream Identifier. (present if Number of Entries > 0)
```